pyiron_base/master/generic.py

```diff
@@ -440,16 +441,28 @@ def __getitem__(self, item):
 
     def __getattr__(self, item):
         """
-        CHeck if a job with the specific name exists
+        Check if a job with the specific name exists
 
         Args:
             item (str): name of the job
 
         Returns:
 
         """
-        item_from_get_item = self.__getitem__(item=item)
-        if item_from_get_item is not None:
+        exception = False
```

Comment:
Is this related to this change or does it just pop up now on your machine?

Reply:
Yes, the following code from a `unittest` causes a `RecursionError`, if the now renamed `copy_self` method is called, without the exception part that I added:

```
child = pr.create_job( job_type=GenericJob,  job_name='child' )
master = pr.create_job(job_type=GenericMaster, job_name='master')
master.append(child)
master_copy = master.copy_to(project=pr, new_job_name='copy', new_database_entry=False)
```

Reply:
I guess this is what [this](https://github.com/pyiron/pyiron_base/blob/232a144847e0ba1223eacaa6d76ac17e153c7b61/pyiron_base/job/generic.py#L458) is supposed to mean.  In another way it means someone owes us a beer.

Reply:
I'm gonna add the `code smell` tag now... :P